presto-main/src/main/java/com/facebook/presto/sql/gen/InCodeGenerator.java

```diff
@@ -126,21 +128,20 @@ public ByteCodeNode generateExpression(Signature signature, ByteCodeGeneratorCon
                     .append(switchCaseBlocks);
         }
         else {
-            // TODO: replace Set with fastutils (or similar) primitive sets if types are primitive
-            // for huge IN lists, use a Set
-            Binding constant = generatorContext.getCallSiteBinder().bind(constantValues, Set.class);
+            // for huge IN lists, use a Fastutil primitive Set
+            @SuppressWarnings("unchecked")
+            Set<?> fastutilConstantValuesSet = toFastutilHashSet(constantValues, (Class) javaType);
```

Comment:
All these usages of `Class` should be `Class<?>` since `Class` is a parameterized class (enable the IDE warning about raw generics usage)